demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -475,6 +483,48 @@ export class DemoMeetingApp
       }
   }
 
+  async createReplacementImageBlob(startColor: string, endColor: string): Promise<Blob> {
+    const canvas = document.createElement("canvas");
+    canvas.width = 500; 
+    canvas.height = 500;
+    var ctx = canvas.getContext("2d");
+    var grd = ctx.createLinearGradient(0, 0, 250, 0);
+    grd.addColorStop(0, startColor);
+    grd.addColorStop(1, endColor);
+    ctx.fillStyle = grd;
+    ctx.fillRect(0, 0, 500, 500);
+    const blob = await new Promise<Blob> (resolve => {
+      canvas.toBlob(resolve);
+    });
+    return blob;
+  }
+
+  /**
+  * The image blob in this demo is created by generating an image
+  * from a canvas, but another common scenario would be to provide 
+  * an image blob from fetching a URL.
+  *   const image = await fetch('https://someimage.jpeg');
+  *   const imageBlob = await image.blob();
+  */
+  async getBackgroundReplacementOptions(): Promise<BackgroundReplacementOptions> {
```

Comment:
Non-blocking nit: Would be nice to consolidate all the background blur/transform device/video filter into a separate file. Lessens size of `meetingV2.ts` but more importantly IMO it makes it easier for builders with existing applications to find reference (in addition to the guide) and possible copy pastable snippets to bring in video filters, etc. into application.

Also @ltrung do we have filters in react demo? I wonder if it makes sense to make some sort of sdk-demo-common package ;P.

Reply:
Makes sense. I will add a task to consolidate background filter code into a separate file. 

Also, we do have blur available in react demo and we will have replacement added as well.